.claude/skills/investigate-issue/SKILL.md

```
@@ -0,0 +1,50 @@
+---
+name: investigate-issue
+description: Investigate a Questarr GitHub issue by pulling the linked log entry from the private Doezer/Questarr-logs repo and cross-referencing it against the codebase
+disable-model-invocation: false
+---
+
+Questarr users can submit scrubbed server logs from the in-app "Send Logs" dialog
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)